docs/configuration.md

```diff
@@ -0,0 +1,139 @@
+Configurating lookout
+===
+
+Global server configuration is stored in `config.yml` at the root directory of **lookout**. You can ask `lookoutd` to load any other config file with `--config=PATH_TO_FILE`.
+
+The main structure of `config.yml` is:
+
+```yml
+providers:
+    github:
+        # configuration of GitHub provider
+repositories:
+    # list of repositories to watch and user/token if needed
+analyzers:
+    # list of named analyzers
+```
+
+
+# Github provider
+
+The `providers.github` key configures how **lookout** will connect with GitHub. 
+
+```yml
+providers:
+  github:
+    comment_footer: "_If you have feedback about this comment, please, [tell us](%s)._"
+    # app_id: 1234
+    # private_key: ./key.pem
+    # installation_sync_interval: 1h
+```
+
+`comment_footer` key defines a format-string that will be used for custom messages for every message posted on GitHub; see how to [add a custom message to the posted comments](#custom-footer)
+
+<a id=basic-auth></a>
```

Comment:
`<a id=basic-auth></a>` maybe it shouldn't be here...

Reply:
I do not like to use anchors using the autogenerated ids because they can be easily broken if the title is eventually changed.
But if you really prefer to go with autogenerated ones, just confirm it and I'll use them :+1:

Reply:
Nah. Up to you. It was just a suggestion.